docs/api-guide/fields.md

```diff
@@ -302,6 +302,15 @@ Corresponds to `django.db.models.fields.TimeField`
 
 Format strings may either be [Python strftime formats][strftime] which explicitly specify the format, or the special string `'iso-8601'`, which indicates that [ISO 8601][iso8601] style times should be used. (eg `'12:34:56.000000'`)
 
+## DurationField
+
+A Duration representation.
+Corresponds to `django.db.models.fields.Duration` for Django>=1.8
+otherwise to `django.db.models.fields.BigIntegerField`.
```

Comment:
@tomchristie I tried to find a better place for the documentation but I couldn't find it.
Please, Can you explain in details what you expect ?